ui/src/components/modals/DeleteModelModal.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright 2020 The Merlin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from "react";
+import { EuiConfirmModal, EuiFieldText, EuiOverlayMask, EuiProgress } from "@elastic/eui";
+import { useMerlinApi } from "../../hooks/useMerlinApi";
+import mocks from "../../mocks";
+
+const DeleteModelModal = ({
+  model,
+  callback,
+  closeModal
+}) => {
+  const [activeEndpoint, setActiveEndpoint] = useState([])
```

Comment:
Nit: Perhaps `activeEndpoints` and `setActiveEndpoints` (in plural) are more accurate.